app/src/main/java/org/eyeseetea/malariacare/domain/entity/OrgUnit.java

```diff
@@ -0,0 +1,53 @@
+package org.eyeseetea.malariacare.domain.entity;
+
+
+public class OrgUnit {
```

Comment:
@manuelplazaspalacio After @ifoche merge this PR https://github.com/EyeSeeTea/pictureapp/pull/1074 There is already a domain entity OrganisationUnit with similar logic, can you refactor for unify code?